pkg/descheduler/apis/config/validation/validation_loadaware.go

```diff
@@ -34,12 +34,6 @@ func ValidateLowLoadUtilizationArgs(path *field.Path, args *deschedulerconfig.Lo
 		allErrs = append(allErrs, field.Invalid(path.Child("evictableNamespaces"), args.EvictableNamespaces, "only one of Include/Exclude namespaces can be set"))
 	}
 
-	if args.NodeSelector != nil {
```

Comment:
how about we just don't allow set NodeSelector and NodePools at the same time.
we can treat all nodes as one "pool" so that the NodeSelector config still works.

Reply:
At present, it supports automatic conversion of the original nodeSelector to nodePool